sw/device/sca/ecc_serial.c

```diff
@@ -157,33 +203,34 @@ static void p256_ecdsa_sign(const uint32_t *msg, const uint32_t *private_key_d,
                             uint32_t *signature_r, uint32_t *signature_s,
                             const uint32_t *k) {
   uint32_t mode = 1;  // mode 1 => sign
-  LOG_INFO("Copy data");
+  // LOG_INFO("Copy data");
```

Comment:
If I understand correctly, you're "removing" these `LOG_INFO()` calls to have exactly equal timing for many iterations and I see that this is necessary. However, I would either completely remove those calls or alternatively gate them with another variable or a `DEFINE` or something that allows us to quickly re-enable them for debugging purposes. WDYT?

Reply:
This seems to have been addressed now.